### PR TITLE
Add `Community Contributors 2023: Elite Nominators` news post

### PR DESCRIPTION
**Pull request by Hiviexd**
# please do not spoil the results, especially to the recipients

- [x] awards

### CODE REVIEW

File: news/2024/2024-02-28-community-contributors-elite-nominators-2023.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+[![](/wiki/shared/news/2024-02-28-community-contributors-elite-nominators-2023/FuJu.jpg)](https://osu.ppy.sh/users/10773882)
+
+With standards and immense contributions to the Ranked scene, [FuJu](https://osu.ppy.sh/users/10773882) has again proven himself a prudent nominator in 2023. He promoted various map styles, mappers and song genres last year, with 65 nominations and only one (1) notable reset. Moreover, as an NAT member, he has evaluated nearly all of the disqualifications that happened last year. He has also been actively involved in the BN mentorship program, helping potential BN candidates to improve and understand how to become a competent modder.
```
Comment (0x84f) [inline news/2024/2024-02-28-community-contributors-elite-nominators-2023.md:34]:
```suggestion
With standards and immense contributions to the Ranked scene, [FuJu](https://osu.ppy.sh/users/10773882) has again proven himself a prudent nominator in 2023. He promoted various map styles, mappers and song genres last year, with 65 nominations and only one (1) notable reset. Moreover, as an NAT member, he has evaluated nearly all the disqualifications that happened last year. He has also been actively involved in the BN mentorship program, helping potential BN candidates to improve and understand how to become a competent modder.
```

File: news/2024/2024-02-28-community-contributors-elite-nominators-2023.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+Finishing strong in 2023, AWMRone has set a prime example in the high-level osu!mania 7K scene with his outstanding performance, so let's look forward to what he can bring us in 2024!
+
+[![](/wiki/shared/news/2024-02-28-community-contributors-elite-nominators-2023/Furryswan.jpg)](https://osu.ppy.sh/users/9555243)
+
+Delivering to you, one of our longstanding osu!mania BNs, [Furryswan](https://osu.ppy.sh/users/9555243) is one of the few nomintators who have maintained consistent performance throughout their tenures while at the same time staying true to their passion of nominating a lot of technical and uncommon styles of 4K, featuring graces and niche patterning that are rare to see in Ranked nowadays. The breakthrough and novelty to achieve this has consistently been made for years and it's finally time for him to be recognised for his contribution.
```
Comment (0x84f) [inline news/2024/2024-02-28-community-contributors-elite-nominators-2023.md:136]:
```suggestion
Delivering to you, one of our longstanding osu!mania BNs, [Furryswan](https://osu.ppy.sh/users/9555243) is one of the few nomintators who have maintained consistent performance throughout their tenures while at the same time staying true to their passion of nominating a lot of technical and uncommon styles of 4K, featuring graces and niche patterning that are rare to see in Ranked nowadays. The breakthrough and novelty to achieve this has consistently been made for years, and it's finally time for him to be recognised for his contribution.
```

File: news/2024/2024-02-28-community-contributors-elite-nominators-2023.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+He has not only pushed sets that distinctly separate himself from the other BNs, but his modding quality and the amount of detail he puts into all of his mods shows his expertise in handling each map with the highest degree of quality. This doesn't happen often and can only be done through the sheer tenacity of learning and experiencing this unconventional skill for years.
+
+It's inspiring to see him thrive and we hope that his achievement will also be a source of inspiration for the other BNs, signaling that diligence and perseverance do pay off when working for consistency and aiming to be the best.
```
Comment (0x84f) [inline news/2024/2024-02-28-community-contributors-elite-nominators-2023.md:140]:
```suggestion
It's inspiring to see him thrive, and we hope that his achievement will also be a source of inspiration for the other BNs, signaling that diligence and perseverance do pay off when working for consistency and aiming to be the best.
```

File: news/2024/2024-02-28-community-contributors-elite-nominators-2023.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+
+[![](/wiki/shared/news/2024-02-28-community-contributors-elite-nominators-2023/Protastic101.jpg)](https://osu.ppy.sh/users/6712747)
+
+Avid fan of 5K and keysounds, [Protastic101](https://osu.ppy.sh/users/6712747) has over the last year put in a lot of work and effort and managed to nominate a total of 134 beatmaps, mostly from 5K maps as well as newer mappers creating their first Ranked map.
+
+They also did a lot outside of just the regular BN work, putting in a big effort in the mania modding hub where they have organised a lot of events for new mappers to get feedback and join the community. It cannot be understated how important that is for the mapping scene.
```
Comment (0x84f) [inline news/2024/2024-02-28-community-contributors-elite-nominators-2023.md:146]:
```suggestion
They also did a lot outside just the regular BN work, putting in a big effort in the mania modding hub where they have organised a lot of events for new mappers to get feedback and join the community. It cannot be understated how important that is for the mapping scene.
```

**Review by 0x84f — COMMENTED** [review]
small stuff

**Review by Walavouchey — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**